packages/Loader/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { forwardRef } from 'react'
+import { oneOf, string } from 'prop-types'
+
+import * as S from './styles'
+
+export const Loader = forwardRef(({ color, dataTestId, size = 'sm', ...rest }, ref) => (
+  <S.LoaderWrapper data-testid={dataTestId} ref={ref} {...rest}>
+    <S.LoadingDot backgroundColor={color} shape="circle" size={size} />
```

Comment:
add a color by default on theme, and add the possibilty to pass a width and height props if we want to put another size custom ?

Reply:
the size is configured with the theme so you can create your own width/height for any size

Reply:
yes i know, but we need to do the same everywhere and for exemple an icon can have a size prop or width and height for special case